blockmanager.go

```diff
@@ -352,6 +357,13 @@ func (b *blockManager) resetHeaderState(newestHash *chainhash.Hash, newestHeight
 	}
 }
 
+// SyncHeight returns the current synced block height from peers.
```

Comment:
I'd reword this slightly, because reading it, it seems that this is the block height the local node *has* synced, while this is meant to be the block height the local node is trying to sync *to* (ie: the "target" sync height). The semantics for this "syncHeight" are also slighly tricky, since it's either the current best height (on a node that has just loaded up and hasn't tried syncing to anything yet or that has lost all peers) _or_ it's the height it's trying to sync to.

This would also be useful information returned in `getblockchaininfo`, since this rpc already seems to include info that is not strictly from the node's chain.